dvc/command/experiments.py

```diff
@@ -381,16 +395,22 @@ def show_experiments(
     if no_timestamp:
         td.drop("Created")
 
+    for col in ("State", "Executor"):
```

Comment:
Is it possible for other columns to be empty? If so, would we want to hide them, too?

Reply:
There shouldn't be any other columns in the table right now that can be empty